js/util/lru_cache.js

```diff
@@ -111,6 +111,23 @@ class LRUCache<T> {
     }
 
     /**
+     * Remove a key/value combination from the cache.
+     *
+     * @param {string} key the key for the pair to delete
+     * @returns {LRUCache} this cache
+     * @private
+     */
+    remove(key: string) {
+        if (!this.has(key)) { return this; }
+
+        this.onRemove(this.data[key]);
```

Comment:
Yeah, fair enough.  I'd actually be okay with "returning expired tiles to nowhere" (i.e. the calling code need not actually store the return value), BUT because it's called `get` I agree it would look pretty weird:

```js
// oh, we need to remove the cache entry now, so
cache.get(id)
// cool, moving on
```

I sort of wonder if we should just _rename_ `LRUCache#get` to `LRUCache#remove` -- that way, whether the calling code actually uses the return value or not, it is very clear that you're removing the entry from the cache.